dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -51,6 +56,11 @@ public Node getArgument(int i) {
         return arguments.get(i);
     }
 
+    public Node getReceiver() {
```

Comment:
We should be more careful with purity of all our node classes. Can you add `@Pure` to all the getters in this class? (And then probably add some javadoc too.